lib/runner/cli/nightwatch.conf.ejs

```diff
@@ -253,6 +253,61 @@ module.exports = {
       }
     },
     //////////////////////////////////////////////////////////////////////////////////
+    // Configuration for when using the SauceLabs cloud service                      |
+    //                                                                               |
+    // Please set the username and access key by setting the environment variables:  |
+    // - SAUCE_USERNAME                                                              |
+    // - SAUCE_ACCESS_KEY                                                            |
+    //////////////////////////////////////////////////////////////////////////////////
+    sauce: {
+      selenium: {
+        host: 'ondemand.saucelabs.com',
+        port: 443
+      },
+      // More info on configuring capabilities can be found on:
+      // https://wiki.saucelabs.com/display/DOCS/Test+Configuration+Options
+      desiredCapabilities: {
+        'sauce:options' : {
+          username: '${SAUCE_USERNAME}',
+          access_key: '${SAUCE_ACCESS_KEY}',
```

Comment:
I think we should rename this to `SAUCE_USER` and `SAUCE_KEY` as for browserstack we have `BROWSERSTACK_USER` and `BROWSERSTACK_KEY`

Reply:
I had the same thought, but SauceLabs has a different naming convention so users would already have them created per their guidelines https://docs.saucelabs.com/basics/environment-variables/ . I don't think we should make users created multiple entries.